### PR TITLE
Add docdb, neptune, mariadb, mysql, oracle, redshift

### PR DESCRIPTION
**Pull request by mattJsonar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+### Features
+- Amazon DocumentDB module
+- Amazon Neptune module
+- Amazon Neptune Slow query module
+- Amazon RDS MariaDB module
+- Amazon RDS MySQL module
+- Amazon RDS MySQL Slow query module
+- Amazon RDS Oracle (standard auditing) module
+- Amazon RDS Oracle (unified auditing) module
+- Amazon Redshift module
```
Comment (PetalJsonar) [inline CHANGELOG.md:14]: Do we want to specify both ODBC and S3 versions for Redshift? if we're already differentiating between regular and slow query modules and the different auditing types for RDS Oracle

File: DSF_VERSION_COMPATABILITY.md [hunk 2]
```diff
@@ -7,9 +7,45 @@ The following table lists the DSF versions that each module is tested and mainta
    <th>DSF Versions</th>
   </tr>
```
Comment (PetalJsonar) [inline DSF_VERSION_COMPATABILITY.md:9]: I think this table is missing Redshift

File: examples/onboard-aws-docdb-cluster/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+## Providers
+
+No providers.
+
+## Modules
+
+| Name | Source | Version |
+|------|--------|---------|
+| <a name="module_aws-default-account-asset"></a> [aws-default-account-asset](#module\_aws-default-account-asset) | imperva/agentless-onboarding/dsfhub//modules/dsfhub-aws-cloud-account | n/a |
```
Comment (PetalJsonar) [inline examples/onboard-aws-docdb-cluster/README.md:25]: > imperva/agentless-onboarding/dsfhub//modules/
Extra slash here?

File: DSF_VERSION_COMPATABILITY.md (hunk 2 above)
Comment (mattJsonar) [reply DSF_VERSION_COMPATABILITY.md:9]: Damn you copy-paste

File: examples/onboard-aws-docdb-cluster/README.md (hunk 3 above)
Comment (mattJsonar) [reply examples/onboard-aws-docdb-cluster/README.md:25]: Looks weird but it's intentional. `imperva/agentless-onboarding/dsfhub//` tells terraform where to look in the registry, and then `modules/dsfhub-aws-cloud-account` is the path to follow from there

File: examples/onboard-aws-redshift/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+This example includes additional prerequisites that will need to be completed to fully utilize the module. More details can be found in the [onboarding documentation](https://docs.imperva.com/bundle/onboarding-databases-to-sonar-reference-guide/page/Amazon-Redshift-Onboarding-Steps_48367115.html).
+
+It creates both 'aws' and 'dsfhub' resources. More information regarding authentication to each one can be found in the relevant provider documentation:
+- [aws](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)
+- [dsfhub](https://registry.terraform.io/providers/imperva/dsfhub/latest/docs)
+
+Importantly, the example includes two different audit methods: auditing via the Audit Tables, or auditing via an S3 bucket. <b>Please choose your desired audit method</b>.
+
+## Option 1: Auditing via Audit Tables
+## Prerequisites
```
Comment (PetalJsonar) [inline examples/onboard-aws-redshift/README.md:11]: I fix these in my PR in the private repo, but this and `Database Configuration` should be a `###` and same thing with the subheadings under Option 2

**Review by PetalJsonar — APPROVED** [review]

**Pull request merged**